pkg/configuration/client.go

```diff
@@ -62,34 +63,34 @@ func (c capsuleConfiguration) ForceTenantPrefix() bool {
 	return c.retrievalFn().Spec.ForceTenantPrefix
 }
 
-func (c capsuleConfiguration) CASecretName() (name string) {
-	name = CASecretName
+func (c capsuleConfiguration) TLSSecretName() (name string) {
+	name = TLSSecretName
 
 	if c.retrievalFn().Annotations == nil {
 		return
 	}
 
-	v, ok := c.retrievalFn().Annotations[capsulev1alpha1.CASecretNameAnnotation]
+	v, ok := c.retrievalFn().Annotations[capsulev1alpha1.TLSSecretNameAnnotation]
```

Comment:
I see that the annotation `CASecretNameAnnotation` is not more used, isn't it?

Formerly, it was used to specify a different Secret name that was containing the CA.

Should we drop it entirely with this refactoring, since we're going to have a single Secret resource containing all the required keys?

Reply:
Just saw the code, and yes, it is no more used: however, we're still mentioning it in several markdown documents for docs.

I'm going to create a new issue for the documentation, since this is still valid with Capsule v0.1.1, drop it from the code-base and everything should be set.

Reply:
@prometherion do i need to remove it from
https://github.com/clastix/capsule/blob/60e826dc832c59b16a3638b161230459681f3a3f/api/v1alpha1/capsuleconfiguration_annotations.go#L8?